lib/foreman_leapp/engine.rb

```diff
@@ -23,31 +23,27 @@ class Engine < ::Rails::Engine
 
         # add dashboard widget
         widget 'foreman_leapp_widget', name: N_('Foreman plugin template widget'), sizex: 4, sizey: 1
+
+        # make sure is available to the foreman
+        register_global_js_file 'fills'
+
+        extend_page("job_invocations/show") do |context|
+          context.add_pagelet :main_tabs,
+            :name => _("LEAPP"),
+            :partial => "job_invocations/leapp_tab",
+            :onlyif => proc { |job_invocation| true }
+        end
       end
     end
 
     # Include concerns in this config.to_prepare block
     config.to_prepare do
       begin
         Host::Managed.include ForemanLeapp::HostExtensions
-        HostsHelper.prepend ForemanLeapp::HostsHelperExtensions
+        HostsHelper.include ForemanLeapp::HostsHelperExtensions
```

Comment:
I do not see any changes in `ForeamnLeapp::HostsHelperExtensions`, why changing it to `include`?